### PR TITLE
Add ETH price

### PR DESCRIPTION
**Pull request by paszkowskiDamian**

### CODE REVIEW

File: src/types/history.ts [hunk 1]
```diff
@@ -19,6 +20,7 @@ interface DepositEvent extends HistoryEventBase {
   dai_amount: string;
   collateral_amount: string;
   oracle_price: string;
+  eth_price: string;
```
Comment (johnnieskywalker) [inline src/types/history.ts:23]: Not sure if string is the best type. Guess it is to store BigNumber in Postgres and convert it in both ways. https://stackoverflow.com/questions/46094733/postgres-and-javascript-require-to-support-more-than-15-digits-including-decimal

Comment (paszkowskiDamian) [reply src/types/history.ts:23]: This is what db client returns, in schema it is `decimal (72,18)`

**Comment by johnnieskywalker** [comment]
After testing couple of queries ethPrice is working 
image

**Comment by johnnieskywalker** [comment]
also for move events prices are in place 
image

**Review by johnnieskywalker — APPROVED** [review]

**Pull request merged**